File: app/src/at-nhatnguyen/java/asiantech/internship/summer/javacore/Exercise3.java
package asiantech.internship.summer.javacore;
import java.util.Scanner;
public class Exercise3 {
    private static int surplus, sum;
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        System.out.println("Nhap day so nguyen:");
        int a = scan.nextInt();
        while (a > 0) {
            surplus = a % 10;
            a = a / 10;
            sum += surplus;
        }
        System.out.println("Tong cac so la:"+ sum);
    }
}

File: app/src/at-nhatnguyen/java/asiantech/internship/summer/kotlin/Exercise3.kt
package asiantech.internship.summer.kotlin

import java.util.Scanner

fun main() {
    var surplus: Int
    var sum = 0
    val scan = Scanner(System.`in`)
    print("Nhap day so nguyen: ")
    var a = scan.nextInt()
    while (a > 0) {
        surplus = a % 10
        a /= 10
        sum += surplus
    }
    println("Tong cac so la:$sum")
}
